docs/doc_2_error_handling.md

```diff
@@ -42,3 +42,21 @@ if ok && apiError.GetError() == 404 {
  // Do something
 }
 ```
+
+## Mocking Errors
+
+SDK errors can be mocked by creating instance of `GenericOpenAPIError` struct. 
+
+```go
```

Comment:
let's add the same comment on surrounding code omitted ;)

Reply:
`// Surrounding code omitted for brevity` - I see now. Adding